src/main.ts

```diff
@@ -2,19 +2,27 @@ import { NestFactory } from '@nestjs/core';
 import { ValidationPipe } from '@nestjs/common';
 
 import { TransformInterceptor } from '@common/interceptors/transform.interceptor';
+import { SwaggerModule } from '@nestjs/swagger';
+
+import { swaggerConfig, validationPipeOptions } from '@common/config';
 import { AppModule } from './app.module';
 
+enum ApiVersion {
+  V1 = 'api/v1',
+}
+
+const DOCUMENTATION_PATH = `${ApiVersion.V1}/docs`;
```

Comment:
Could we move the DOCUMENTATION_PATH and ApiVersion away from main? Maybe move it to constants

Reply:
yes, sure